test/test_linalg.py

```diff
@@ -945,7 +945,7 @@ def get_value_type(dtype):
             else:
                 self.assertEqual(result.dtype, from_dtype, msg=msg)
 
-            result_out = torch.empty((), dtype=to_dtype, device=device)
+            result_out = torch.empty((0), dtype=to_dtype, device=device)
```

Comment:
What's up with this change?

Reply:
It works fine either way, but I was getting this `resize_output` warning with size `()`, so I figured it's better to use `(0)`:

```
UserWarning: An output with one or more elements was resized since it had shape [], which does not match the required output shape [1, 1].This behavior is deprecated, and in a future PyTorch release outputs will not be resized unless they have zero elements. You can explicitly reuse an out tensor t by resizing it, inplace, to zero elements with t.resize_(0).
```